docker/build-gstreamer/compile

```diff
@@ -13,6 +13,9 @@
 set -ex
 
 for repo in gstreamer libnice gst-plugins-base gst-plugins-bad gst-plugins-good gst-plugins-ugly gst-libav gst-rtsp-server; do
```

Comment:
Since 1.20, all components are built from the GStreamer mono repo, no need to build them one by one